addons/arsenal/XEH_postInit.sqf

```diff
@@ -81,25 +79,46 @@ GVAR(lastSortRight) = "";
         _contentPanelCtrl lnbSort [1, false];
 
         // Select previously selected loadout
-        for '_i' from 0 to (((lnbsize _contentPanelCtrl) select 0) - 1) do {
-            if ((_contentPanelCtrl lnbText [_i, 1]) == _curSelData) exitwith {_contentPanelCtrl lnbSetCurSelRow _i};
+        for "_lbIndex" from 0 to (lnbSize _contentPanelCtrl select 0) - 1 do {
+            if ((_contentPanelCtrl lnbText [_lbIndex, 1]) == _curSelData) exitWith {
+                _contentPanelCtrl lnbSetCurSelRow _lbIndex
+            };
         };
     };
 }] call CBA_fnc_addEventHandler;
 
 ["CBA_loadoutSet", {
     params ["_unit", "_loadout", "_extendedInfo"];
+
+    // Set face
     private _face = _extendedInfo getOrDefault [QGVAR(face), ""];
+
     if (_face != "") then {
-        _unit setFace _face;
+        if (isMultiplayer) then {
+            private _id = [QGVAR(broadcastFace), [_unit, _face], QGVAR(centerFace_) + netId _unit] call CBA_fnc_globalEventJIP;
```

Comment:
I'm not a huge fan of setting the face every time `"CBA_loadoutSet"` is called, but I don't think there is a way around it.
On a more important note: Is `netId` the right command for the job? What this needs is a constant ID for the same unit (and I mean object, which means after respawning it must change).

Reply:
netId is specific to the object, so yes should be good.
Alternatively you could use hashValue of the object (but that also uses netId internally)

Important, keep the netId issues in singleplayer in mind, see wiki page

Reply:
BIS_fnc_netId doe the same as netID but is fully SP compatible. netID does return nothing/nil in SP.